tests/h/util/group_test.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import pytest
+
+from h.util import group as group_util
+
+
+class TestSplitGroupID(object):
+    @pytest.mark.parametrize('groupid,authority_provided_id,authority', [
+        ('group:flashbang@dingdong.com', 'flashbang', 'dingdong.com'),
+        ('group::ffff@dingdong.com', ':ffff', 'dingdong.com'),
+        ('group:\\f!ork/ßlift@sprongle.co', '\\f!ork/ßlift', 'sprongle.co'),
```

Comment:
I realize now that this value for `authority_provided_id` would not be allowed by the model (I wrote this test before imposing the restrictions in the model on this field), but the _test_ is still valid, I think.